src/utils/handler.ts

```diff
@@ -18,6 +19,7 @@ export function wrap<TEvent, TResult>(
   return async (event: TEvent, context: Context) => {
     try {
       await onStart(event, context);
+      incrementInvocationsMetric(context.invokedFunctionArn);
```

Comment:
The incrementInvocationsMetric/incrementErrorsMetric  logic should probably be  moved inside the onStart/onComplete handlers. The idea behind this class is that it's a utility that could be moved out of dd-lambda-js, and potentially be used to wrap one handler multiple times. onComplete can be modified to accept an result or error object.

Reply:
Also, do you think we should make this an opt-in config settings? I'm a little worried about customer's accidentally adding synchronous metric calls to their lambdas.